apps/web/declarations/environment.d.ts

```diff
@@ -80,6 +80,25 @@ declare global {
        * Client secret of the oidc provider
        */
       OIDC_CLIENT_SECRET?: string;
+      /**
+       * Scopes that should be used when configuring the oidc provider
+       */
+      OIDC_SCOPES?: string;
+      /**
+       * If Rallly should expect the oidc provider to return an ID token
+       */
+      OIDC_ID_TOKEN_EXPECTED?: boolean;
```

Comment:
Currently environment variables are not parsed so they will all be strings.

Reply:
I didn't know, thanks for the information. I changed the two variables that were previously boolean.